src/main/java/org/gradlex/javaecosystem/capabilities/rules/CapabilityDefinitions.java

```diff
@@ -17,32 +17,7 @@
 package org.gradlex.javaecosystem.capabilities.rules;
 
 import org.gradlex.javaecosystem.capabilities.resolution.DefaultResolutionStrategy;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaActivationApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaActivationImplementationRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaAnnotationApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaMailApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaServletApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaWebsocketApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaWebsocketClientApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JakartaWsRsApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxActivationApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxAnnotationApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxEjbApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxElApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxInjectApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxJsonApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxJwsApisRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxMailApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxPersistenceApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxServletApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxServletJspRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxServletJstlRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxSoapApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxValidationApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxWebsocketApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxWsRsApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxXmlBindApiRule;
-import org.gradlex.javaecosystem.capabilities.rules.jakarta.JavaxXmlWsApiRule;
+import org.gradlex.javaecosystem.capabilities.rules.jakarta.*;
```

Comment:
Can you please keep the import list as it is.

Reply:
Sure, but why, though?

Idea did that automatically as soon as I wrote the just-created Jakarta rule